README.md

```diff
@@ -143,6 +143,13 @@ Instead of *just* looking for `.goldengrahamsrc` (no extension), it will also lo
 - `.goldengrahamsrc.yml`
 - `.goldengrahamsrc.js`
 
+##### startDir
```

Comment:
Why do you think it's preferable to add this new option instead of having the `searchPath` argument default to `process.cwd()`?

Reply:
Mostly for symmetry with `configPath`. Can change it if you feel that'd be better.

Reply:
Yeah, I think a default value for `searchPath` makes just as much sense here while avoiding an addition to the API. Unless you feel differently, I'd prefer that approach.

Reply:
Agreed and done.